src/requests/graphql/example.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from '@apollo/client';
+import client from '../../utils/apollo-client';
+
+import { Country } from '../../interfaces/country';
+
+export const getCountries = async (): Promise<Country[]> => {
+    const { data } = await client.query({
+        query: gql`
+            query Countries {
+                countries(filter: { currency: { eq: "USD" } }) {
+                    name
+                    code
+                    emoji
+                }
```

Comment:
You should be able to call rest API as well , without waiting for graphql layer to be ready. We are not sure if we want to add graphql or not at this point correct.

Reply:
Documentation : https://www.apollographql.com/docs/react/api/link/apollo-link-rest/

Reply:
Yeah we could hit a rest API, I kinda assumed that since we're implementing Apollo that we are going to go in the GraphQL direction, but I'm also open to just using rest endpoints if we want to write those on lambda functions.

Reply:
Agreed - but I don't want us to wait for graphql layer to be ready. Also we donno if we get any benefit by adding Graphql to our backend. Why code when we don't have necessity :). We can start with strapi end points which we already have and then code new backend API's as we go in the best way we decide.

Reply:
Ahh ok that makes sense to me. In the meantime, should we add that `apollo-link-rest`, or just fetch normally?